internal/web_server/web/views/pages/quiz_pages/quiz_summary_page.templ

```diff
@@ -64,7 +64,7 @@ templ QuizSummaryContent(summary *user_quiz_summary.UserQuizSummary) {
 		<section class="flex flex-col gap-4 items-center scrollbar-hide">
 			<h2 class="text-2xl">Oppsummering</h2>
 			<div class="gradient-outline w-fit h-fit">
-				<ol class="list-decimal pl-8 pr-4 py-4 space-y-5 max-h-[calc(100dvh-24rem)] min-w-80 max-w-full overflow-y-auto scrollbar-hide bg-white rounded-card">
+				<ol class="list-decimal pl-8 pr-4 py-4 space-y-5 max-h-[calc(100dvh-25rem)] min-w-80 max-w-full overflow-y-auto scrollbar-hide bg-white rounded-card">
```

Comment:
This one might need to be more that 25rem. We do `size-28`, `size-36` and `size-44` for the points. It will overflow then.